extensions/resource-deployment/src/ui/notebookWizard/notebookWizardToolsAndEulaPage.ts

```diff
@@ -0,0 +1,516 @@
+					level: azdata.window.MessageLevel.Error
+				};
+				return false;
+			}
+
+			return true;
+		});
+	}
+
+	public initialize(): void {
+		if (this._isInitialized) {
+			return;
+		}
+
+		this.pageObject.registerContent((view: azdata.ModelView) => {
+			this.view = view;
+			const tableWidth = 1126;
+			this._optionsContainer = view.modelBuilder.flexContainer().withLayout({ flexFlow: 'column' }).component();
+			this._agreementContainer = view.modelBuilder.divContainer().component();
+			const toolColumn: azdata.TableColumn = {
+				value: localize('deploymentDialog.toolNameColumnHeader', "Tool"),
+				width: 105
+			};
+			const descriptionColumn: azdata.TableColumn = {
+				value: localize('deploymentDialog.toolDescriptionColumnHeader', "Description"),
+				width: 270
+			};
+			const installStatusColumn: azdata.TableColumn = {
+				value: localize('deploymentDialog.toolStatusColumnHeader', "Status"),
+				width: 70
+			};
+			const versionColumn: azdata.TableColumn = {
+				value: localize('deploymentDialog.toolVersionColumnHeader', "Version"),
+				width: 75
+			};
+			const minVersionColumn: azdata.TableColumn = {
+				value: localize('deploymentDialog.toolMinimumVersionColumnHeader', "Required Version"),
+				width: 105
+			};
+			const installedPathColumn: azdata.TableColumn = {
+				value: localize('deploymentDialog.toolDiscoveredPathColumnHeader', "Discovered Path or Additional Information"),
+				width: 580
+			};
+			this._toolsTable = view.modelBuilder.table().withProperties<azdata.TableComponentProperties>({
+				data: [],
+				columns: [toolColumn, descriptionColumn, installStatusColumn, versionColumn, minVersionColumn, installedPathColumn],
+				width: tableWidth,
+				ariaLabel: localize('deploymentDialog.RequiredToolsTitle', "Required tools")
+			}).component();
+
+			const toolsTableWrapper = view.modelBuilder.divContainer().withLayout({ width: tableWidth }).component();
+			toolsTableWrapper.addItem(this._toolsTable, { CSSStyles: { 'border-left': '1px solid silver', 'border-top': '1px solid silver' } });
+			this._toolsLoadingComponent = view.modelBuilder.loadingComponent().withItem(toolsTableWrapper).withProperties<azdata.LoadingComponentProperties>({
+				loadingCompletedText: localize('deploymentDialog.loadingRequiredToolsCompleted', "Loading required tools information completed"),
+				loadingText: localize('deploymentDialog.loadingRequiredTools', "Loading required tools information"),
+				showText: true
+			}).component();
+
+
+			this._installToolButton = view.modelBuilder.button().withProps({
+				label: localize('deploymentDialog.InstallToolsButton', "Install tools"),
+				CSSStyles: {
+					'display': 'none',
+				},
+				width: '100px',
+			}).component();
+
+			this._installToolButton.onDidClick(() => {
+				this.installTools().catch(error => console.log(error));
+			});
+
+			this.form = view.modelBuilder.formContainer().withFormItems(
+				[
+					{
+						component: this._agreementContainer,
```

Comment:
Make sure you're going through the form for a11y testing to validate that all items are announced correctly and tab navigation works as expected.